hyperspy/model.py

```diff
@@ -835,6 +837,13 @@ def fit(self, fitter=None, method='ls', grad=False,
                 "odr" performs the optimization using the orthogonal distance
                 regression algorithm. It does not support bounds.
 
+                "linear" performs least-squares fitting using linear regression. It does
+                not support bounds, but should be very fast.
+
+                "non_negative_linear" performs least-squares fitting using linear
```

Comment:
"performs least-squares fitting using linear regression" 

perhaps "performs linear least-squares fitting" would be better, since the distinction really is "LLS vs NLLS"